content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

```diff
@@ -206,5 +211,3 @@ The DEB and RPM packages shipped with the Kubernetes releases are:
 | `kubelet`    | Installs the kubelet binary in `/usr/bin` and CNI binaries in `/opt/cni/bin`. |
 | `kubectl`    | Installs the `/usr/bin/kubectl` binary. |
 | `cri-tools` | Installs the `/usr/bin/crictl` binary from the [cri-tools git repository](https://github.com/kubernetes-sigs/cri-tools). |
-
-
```

Comment:
does the file have an empty line at the bottom?
please add one if missing.

Reply:
yes file has an empty line at the bottom